lib/api/model/model.dart

```diff
@@ -570,6 +573,46 @@ enum MessageFlag {
   String toJson() => _$MessageFlagEnumMap[this]!;
 }
 
+/// https://zulip.com/api/get-messages#response
+@JsonSerializable(fieldRename: FieldRename.snake)
+class Submessage {
```

Comment:
Let's put this in the same file as the details of polls, and call it `submessage.dart`.
* That way it's a bit more general than just polls, and makes a natural home for todo lists when we add those in the future.
* Once we have this feature all working, I'd like to avoid thinking about submessages as much as possible. :slightly_smiling_face: And one generally can do that — they don't come up except when specifically working on polls. So it's helpful to have this type not be in between these core message classes that come up all over the Zulip app model.